includes/utils/class-amp-image-dimension-extractor.php

```diff
@@ -109,6 +109,8 @@ public static function normalize_url( $url ) {
 			$url = $home . $path;
 		}
 
```

Comment:
Add a phpdoc comment here that explains what the filter does. For example:

```php
/**
 * Apply filters on the normalized image URL for dimension extraction.
 *
 * @since 1.1
 *
 * @param string $normalized_url Normalized image URL.
 * @param string $url            Original image URL.
 */
```